### PR TITLE
added service instance endpoints and service manifest endpoint

### PR DESCRIPTION
**Pull request by futuregerald**
Added endpoints used in https://github.com/netlify/js-client/blob/master/src/addons.js and https://api.netlify.com/api/v1/services/${addonName}/manifest

### CODE REVIEW

File: swagger.yml [hunk 1]
```diff
@@ -19,7 +19,7 @@ responses:
   error:
     description: error
     schema:
-      $ref: "#/definitions/error"
+      $ref: '#/definitions/error'
```
Comment (bcomnes) [inline swagger.yml]: I'm not opposed to `s/"/'` but lets do it in a separate PR.  Can you unstage these quotation changes?

Comment (futuregerald) [reply swagger.yml]: We're using single quotes everywhere else, are you saying we should leave it as double quotes in this one location?

Comment (futuregerald) [reply swagger.yml]: Done

Comment (bcomnes) [reply swagger.yml]: I'm saying if we fix it, let's do it in a separate PR.

Comment (bcomnes) [reply swagger.yml]: Whitespace/formatting changes are best chunked into a single commit.

Comment (futuregerald) [reply swagger.yml]: Makes sense. Thanks. I fixed it so let me know if it's good to merge.
